packages/grafana-ui/src/components/Menu/MenuItem.tsx

```diff
@@ -53,37 +57,40 @@ export const MenuItem: React.FC<MenuItemProps> = React.memo(
                 }
               : undefined
           }
-          rel={target === '_blank' ? 'noopener noreferrer' : undefined}
+          role={url === undefined ? 'menuitem' : undefined}
+          data-role="menuitem" // used to identify menuitem in Menu.tsx
```

Comment:
Think it's worth making a note about what's going on here.
In general, the "menuitem" role is used to indicate an item within a menu that executes some kind of action upon being selected, like an item in a desktop application's toolbar. Since the MenuItem component can be a link, however, the "menuitem" role makes less sense, and assigning it to a link obscures the fact that clicking on it would navigate the browser to a different page. In light of this, I've chosen to only apply the "menuitem" role when the MenuItem is not a link.
Since we can't rely then on the `role` attribute to identify MenuItems, I've added the "data-role" attribute for that purpose.